src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -21,6 +11,8 @@ description: "Learn how to enable Session Replay in your app if it is not alread
 
 <Include name="beta-note-session-replay.mdx" />
 
+Sentry's Replay product is geared toward developers. By default, it'll mask all fields, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR). Learn more on <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
```

Comment:
@ryan953 Something like this?

Reply:
What are we trying to say by "Sentry's Replay product is geared toward developers"? All of Sentry is developer-focused, so if that's all we mean, this doesn't need to be here. If there's something more specific we want to say about this feature, then we should say that.

```suggestion
By default, it'll mask all fields, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR). Learn more on <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
```